packages/block-library/src/media-text/index.js

```diff
@@ -178,6 +176,7 @@ export const settings = {
 			'is-stacked-on-mobile': isStackedOnMobile,
 			'is-image-fill': imageFill,
 		} );
+		const backgroundStyles = imageFill ? imageFillStyles( mediaUrl, focalPoint ) : {};
```

Comment:
I noticed the focal point picker _doesn't_ have initial values populated on first render (ie: before you click to select any point on the image). Is that intentional? 

When testing with the `cover` Block, I see there _are_ default values applied so it makes me wonder if we're doing it differently for a reason:

![Screen Shot 2019-04-01 at 09 36 03](https://user-images.githubusercontent.com/444434/55314990-39d0ee00-5463-11e9-84a5-e66f40f8f1d5.png)

Reply:
Just pushed an update that fixes an issue with the attribute resets.